src/main/resources/application.conf

```diff
@@ -182,6 +182,12 @@ ergo {
     # which are potentially wallet's. But to be sure, script execution is needed, which could be costly to do in a bulk.
     # So we check from a queue only one box per "scanningInterval".
     scanningInterval = 1s
+
+    # Perform blocks scanning when the wallet is locked (may consume additinal disk space)
```

Comment:
how can we perform block scanning when the wallet is locked?

Reply:
Once the wallet is unlocked addresses to track are loaded to persistent storage, so in the future the wallet is able to save outputs containing these addresses to try to resolve them when the prover is available.